### PR TITLE
Alert setup (via Telegram)

### PR DESCRIPTION
**Pull request by finestructure**
Screenshot 2020-12-02 at 16 36 35

Configured for dev. Remaining prod setup:

- [x] set up telegram bot
- [x] set up token variable
- [x] set up chat id variable

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -125,6 +125,7 @@ services:
     environment:
       GF_SECURITY_ADMIN_PASSWORD: admin
       GF_USERS_ALLOW_SIGN_UP: 'false'
+      GF_SERVER_ROOT_URL: "${GF_SERVER_ROOT_URL}"
```
Comment (finestructure) [inline docker-compose.yml:128]: This allows us to link to dashboards or log views with filter set ups etc. For example the shutdown issue: http://208.52.185.184:3000/goto/mX1lvEAMz

Comment (finestructure) [reply docker-compose.yml:128]: (we may want to set up some domain aliases to avoid having to use the raw IPs)

**Comment by finestructure** [comment]
This is good to go, just leaving it in draft as a reminder for me to set up the prod channel if we're happy with using Telegram.

**Comment by finestructure** [comment]
I should add that we can also set this up with Slack instead. I have no real preference tbh except that it would require us to join yet another workspace 😅

**Review by daveverwer — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (daveverwer) [reply docker-compose.yml:128]: Added aliases on Cloudflare.

**Pull request merged**